P5/Source/Specs/gender.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright TEI Consortium. 
+Dual-licensed under CC-by and BSD2 licences 
+See the file COPYING.txt for details
+$Date$
+$Id$
+-->
+<?xml-model href="https://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="namesdates" ident="gender">
+  <gloss xml:lang="en" versionDate="2022-05-03">gender</gloss>
+  <desc versionDate="2022-05-17" xml:lang="en">specifies the gender identity of a person, persona, or character.</desc>
+  <desc versionDate="2022-05-03" xml:lang="es">especifica la identidad de género de una persona.</desc>
+  <desc versionDate="2022-05-03" xml:lang="it">specifica l'identità di genere di una persona.</desc>
+  <classes>
+    <memberOf key="att.global"/>
+    <memberOf key="model.persStateLike"/>
+    <memberOf key="att.editLike"/>
+    <memberOf key="att.datable"/>
+    <memberOf key="att.typed"/>
+  </classes>
+  <content>
+    <macroRef key="macro.phraseSeq"/>
+  </content>
+  <attList>
+    <attDef ident="type" usage="opt" mode="change">
+      <datatype>
+        <dataRef key="teidata.enumerated"/>
+      </datatype>
+    </attDef>
+    <attDef ident="value" usage="opt">
+      <desc versionDate="2022-05-03" xml:lang="en">supplies a coded value for gender identity</desc>
+      <datatype minOccurs="1" maxOccurs="unbounded"><dataRef key="teidata.gender"/></datatype>
+      <remarks versionDate="2022-05-03" xml:lang="en">
+        <p>Values for this attribute may be locally defined by a project or according to an external standard.</p>
```

Comment:
“Values for this attribute may be taken from an external standard or locally defined by a project”.
Or the wording in comment below.

Reply:
Note that the syntax isn't parallel--revise.

Reply:
Pluralize: external standards.

Reply:
On second thought, don't pluralize. Referring to external standards plural would be making your own local definition after all.

Reply:
Trying this: "Values for this attribute may be defined locally by a project, or they may refer to an external standard."